models/classes/Lis/LisAuthAdapterException.php

```diff
@@ -0,0 +1,29 @@
+<?php declare(strict_types=1);
+/**
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; under version 2
+ * of the License (non-upgradable).
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ *
+ * Copyright (c) 2019 (original work) Open Assessment Technologies SA;
+ */
+namespace oat\taoLti\models\classes\Lis;
+
+use common_user_auth_AuthFailedException;
+
+class LisAuthAdapterException extends common_user_auth_AuthFailedException
+{
+    public function getUserMessage()
+    {
+        return __('Authorization failed');
```

Comment:
let's please check that this message is not something the IMS provides as a default

Reply:
Actually this message isn't used, it's here only because parent abstract `common_user_auth_AuthFailedException` requires me to implement it. 

I catch this exception in the controller and throw new `tao_models_classes_UserException` which is handled by existing `ExceptionInterpretor` (https://github.com/oat-sa/tao-core/blob/a03235fe3f963ac6eb1091b0e1ce30d935378212/models/classes/mvc/error/ExceptionInterpretor.php#L98) which produces a standard response with 403 status code